lib/game_team.rb

```diff
@@ -20,29 +20,70 @@ def self.from_csv(csv_file_path)
   end
 
   def self.home_games
-    (@@all.find_all {|gt| gt.hoa == "home" }).count
+    (all.find_all {|gt| gt.hoa == "home" }).count
   end
 
   def self.percentage_home_wins
-    home_wins = (@@all.find_all {|gt| gt.hoa == "home" && gt.result == "WIN" }).count.to_f
+    home_wins = (all.find_all {|gt| gt.hoa == "home" && gt.result == "WIN" }).count.to_f
     ((home_wins / self.home_games) * 100).round(2)
   end
 
   def self.percentage_visitor_wins
-    visitor_wins = (@@all.find_all {|gt| gt.hoa == "home" && gt.result == "LOSS" }).count.to_f
+    visitor_wins = (all.find_all {|gt| gt.hoa == "home" && gt.result == "LOSS" }).count.to_f
     ((visitor_wins / self.home_games) * 100).round(2)
   end
 
   def self.percentage_ties
-    games_count = @@all.count.to_f
-    ties_count = (@@all.find_all { |gt| gt.result == "TIE"}).count.to_f
+    games_count = all.count.to_f
+    ties_count = (all.find_all { |gt| gt.result == "TIE"}).count.to_f
     ((ties_count / games_count) * 100).round(2)
```

Comment:
the transition from `@@all` to `all` method was completely painless!

Reply:
![image](https://user-images.githubusercontent.com/58446472/79076567-66475900-7cb8-11ea-8042-3ea8b5424ebb.png)